src/FastCache.ts

```diff
@@ -180,12 +180,12 @@ export class FastCache {
 
   //---------------------------------------------------------
 
-  public async withCache(key: string, executor: Promise<any>): Promise<any> {
+  public async withCache(key: string, executor: () => Promise<any>): Promise<any> {
```

Comment:
과할지 모르겠지만 우선 unknown으로 하면 어떨까요?
unknown이 문제되서 나중에 any로 바꿔도 별 지장은 없을 거 같은데

```suggestion
  public async withCache(key: string, executor: () => Promise<unknown>): Promise<unknown> {
```

Reply:
네 반영했습니다